packages/backend/vite.config.js

```diff
@@ -34,6 +34,8 @@ const config = {
       '/@/': join(PACKAGE_ROOT, 'src') + '/',
       '/@gen/': join(PACKAGE_ROOT, 'src-generated') + '/',
       '@shared/': join(PACKAGE_ROOT, '../shared') + '/',
+      '@jsdevtools/ono': '@jsdevtools/ono/cjs/index.js',
+      'ono': '@jsdevtools/ono/cjs/index.js',
```

Comment:
curious why we need that ? these one are usually just for the local source code new links

Reply:
Ono provides both versions, cjs and esm, and vite includes the esm one by default.

But the esm version includes this code, which fails because `module.exports.default` is not defined:
```
// CommonJS default export hack
if (typeof module === "object" && typeof module.exports === "object") {
    module.exports = Object.assign(module.exports.default, module.exports);
}
```

Reply:
shouldn't the extension be set as a module ?

type: "module" ?

I did that for the podman extension
https://github.com/containers/podman-desktop/blob/main/extensions/podman/package.json#L7

and we say we generate a cjs file
https://github.com/containers/podman-desktop/blob/main/extensions/podman/package.json#L13

Reply:
the rationale is that we have more and more libraries that are only esm
so instead of trying to grab the commonjs files, we should switch to be esm and allow to use esm easily as well

some libraries are now esm only
https://github.com/containers/podman-desktop/commit/8e6debfa873a9a55441a3bfb65e87a3652b26ce2

Reply:
@benoitf I just tested with `type: "module"` (and removing the aliases), but the error is the same. The hack in the esm version of ono seems not compatible with `vite`

Reply:
I think that we can't use cjs as for long term it'll be a burden

we may patch the module using https://github.com/JS-DevTools/ono/pull/20 or getting rid of the lines ?

or switch to another library

Reply:
OK, I'll make changes to patch ono's esm version, so we can use it. Thanks @benoitf